lib/pages/settings/view/settings_page.dart

```diff
@@ -96,6 +111,36 @@ class SettingsPageState extends State<SettingsPage> {
                                 context: context),
                           ),
                         ),
+                        const Divider(),
+                        FlatButton(
+                          key: const ValueKey('ask_permissions'),
+                          onPressed: () => {
+                            if (!authProvider.isVerifiedFromCache &&
+                                !authProvider.isAnonymous)
+                              {
+                                AppToast.show(S
+                                    .of(context)
+                                    .messageEmailNotVerifiedAskPermissions)
+                              }
+                            else if (!authProvider.isAnonymous)
```

Comment:
Sooo if the user is anonymous this just does nothing? It's just static text? That's weird.

Whan you can do is add `if (!authProvider.isAnonymous)` before `FlatButton`, that way the option won't show at all for anonymous users (and you can remove the `if` here and just use a simple `else`).